tests/rect_glyf_colr_1_and_picosvg_and_cbdt.ttx

```diff
@@ -72,15 +72,15 @@
   <CBDT>
     <header version="3.0"/>
     <strikedata index="0">
-      <cbdt_bitmap_format_17 name="e000">
+      <cbdt_bitmap_format_17 name="e000.0">
```

Comment:
hm this has the wrong glyph name, it should be "e000" (the base color glyph) not "e000.0" (one of COLR layers)..